README.md

```diff
@@ -44,13 +44,11 @@ For the Docker check to succeed, memory management by cgroup must be enabled on
 On Debian Jessie or later for example you will need to add `cgroup_enable=memory swapaccount=1` to your boot options, otherwise the agent won't be able to recognize your system. See [this thread](https://askubuntu.com/questions/19486/how-do-i-add-a-kernel-boot-parameter/19487#19487) for details.
 
 
-### Service discovery
+### Autodiscovery
 
-The commands we suggest in _Quick Start_ start service discovery in auto config mode. It means the agent will try and detect containers running [some images](https://github.com/DataDog/dd-agent/tree/5.11.2/conf.d/auto_conf) for which we provide a default check that works most of the time.
+The commands in the **Quick Start** section enable Autodiscovery in auto-conf mode, meaning the Agent will automatically run checks against any containers running images listed in the [default check templates](https://github.com/DataDog/dd-agent/tree/5.11.2/conf.d/auto_conf).
```

Comment:
Problem is this folder doesn't exist anymore. It's created at build time, auto_conf files are stored in integrations-core in their related integration folder. I think we should just get rid of the link, or write a KB page that helps locate it on the user machine. Keeping it as is makes it confusing, updating it to point to integrations-core is impractical because users will need to look in each folder individually to find auto_conf files.